test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeUnsafeTests.java

```diff
@@ -121,7 +148,11 @@ static public void testValueHeaderSizeOfClassWithLongField() throws Throwable {
 		}
 	}
 
-	// TODO (#14117): add testing for Unsafe.valueHeaderSize using Unsafe.getObjectSize
+	@Test
+	static public void testValueHeaderSizeIsDifferenceOfGetObjectSizeAndDataSize() {
+		long dataSize = isCompressedRefsEnabled? 8 : 16;
+		assertEquals(myUnsafe.valueHeaderSize(vtPoint.getClass()), myUnsafe.getObjectSize(vtPoint) - dataSize);
```

Comment:
Please also add an assert that `myUnsafe.valueHeaderSize(vtPoint.getClass())` equals to 4 or 8 depending on the compressedrefs mode.

Reply:
addressed in c83a184